app/endpoints/booking.py

```diff
@@ -184,7 +184,7 @@ async def get_bookings_for_manager(
 
 @router.get(
     "/booking/bookings/confirmed",
-    response_model=list[schemas_booking.BookingReturn],
+    response_model=list[schemas_booking.BookingReturnApplicant],
```

Comment:
I'm not sure if we want to return the applicant Phone number and email address

Reply:
I didn't think about that, I'll create a new endpoint for managers

Reply:
Yes I didn't pay attention to that.
I actually don't need to make changes to the API.